doc/support.rst

```diff
@@ -23,44 +23,50 @@ All contributors must agree to the `Contributor Agreement <https://github.com/gl
 Set up a development environment
 --------------------------------
 
-Start by `forking <https://github.com/glotzerlab/signac/fork/>`_ the project.
+1. As a new contributor, start by `forking <https://github.com/glotzerlab/signac/fork>`_ the project.
 
-We highly recommend to setup a dedicated development environment,
-for example with `venv <https://docs.python.org/3/library/venv.html>`_:
+Then clone the repository to your local computer and switch into the repository directory:
 
 .. code-block:: bash
 
-    ~ $ python -m venv ~/envs/signac-dev
-    ~ $ source ~/envs/signac-dev/bin/activate
-    (signac-dev) ~ $ pip install six flake8
+    git clone https://github.com/your-username/signac.git
+    cd signac/
 
-or alternatively with `conda <https://conda.io/docs/>`_:
+2. We highly recommend to setup a dedicated development environment with `conda <https://conda.io/docs/>`_:
 
 .. code-block:: bash
 
-    ~ $ conda create -n signac-dev python=3 six flake8
-    ~ $ activate signac-dev
+    conda create -n signac-dev python --file requirements-dev.txt
+    conda activate signac-dev
```

Comment:
We should give follow up comments regarding how to properly set up the environment.